ocaml/xapi/import.ml

```diff
@@ -234,6 +234,27 @@ let assert_can_live_import __context rpc session_id vm_record =
   if vm_record.API.vM_power_state = `Running || vm_record.API.vM_power_state = `Paused
   then assert_memory_available ()
 
+(* Assert that the local host, which is the host we are live-migrating the VM to,
+ * has free capacity on a PGPU from the given VGPU's GPU group. *)
+let assert_can_live_import_vgpu ~__context vgpu_record =
+  let host = Helpers.get_localhost ~__context in
+  let local_pgpus = Db.PGPU.get_refs_where ~__context ~expr:Db_filter_types.(And
```

Comment:
I find it much nicer that using the same kind of call but serialising the expression by hand (I did that in the past)